lc0/src/utils/transpose.cc

```diff
@@ -34,7 +34,7 @@ void TransposeTensor(const std::vector<int>& dims, std::vector<int> order,
       from_idx += cur_idx[i];
     }
     *to++ = from[from_idx];
-    for (size_t i = dims.size() - 1; i >= 0; --i) {
```

Comment:
With an unsigned int, this would be an infinite loop. I assume this function was never really used then?